ansible/roles/ml-analytics-service/tasks/main.yml

```diff
@@ -42,9 +41,9 @@
     - "{{ WORKDIR }}/faust_as_service"
 
 - name: Change user and create working directory under opt dir and install python virtual environment
-  shell: "cd {{ WORKDIR }} && virtualenv --python=python3.6 spark_venv"
-  become: yes
-  become_user: "{{ USER }}"
+  shell: "cd {{ WORKDIR }} && virtualenv --python=python3.8 spark_venv"
+  become: true
+  become_user: root 
```

Comment:
When you do become: yes it's root user. You don't have to specify that.

Reply:
@rjshrjndrn i will remove that, then

Reply:
done